components/auth/org.wso2.carbon.auth.oauth/src/main/java/org/wso2/carbon/auth/oauth/internal/ServiceReferenceHolder.java

```diff
@@ -126,4 +132,34 @@ public void setConfig(OAuthConfiguration config) {
 
         this.config = config;
     }
+
+    public AuthConfiguration getAuthConfiguration() {
+
+        return authConfiguration;
+    }
+
+    public void setAuthConfiguration(AuthConfiguration authConfiguration) {
+
+        this.authConfiguration = authConfiguration;
+    }
+
+    public PrivateKey getPrivateKey() {
+
+        return privateKey;
+    }
+
+    public void setPrivateKey(PrivateKey privateKey) {
+
+        this.privateKey = privateKey;
+    }
+
+    public Certificate getPublicKey() {
+
+        return publicKey;
+    }
+
+    public void setPublicKey(Certificate publicKey) {
+
```

Comment:
shall we remove these all extra new lines? or is it automatically added by the code formatter?